src/lib/core/CHIPConfig.h

```diff
@@ -2561,6 +2561,15 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_DEVICES 64
 #endif
 
+/**
+ * @def CHIP_CONFIG_CONTROLLER_MAX_ACTIVE_CASE_CLIENTS
+ *
+ * @brief Number of outgoing CASE sessions can be simutaneously negotiated.
```

Comment:
This is a very large number.

Reply:
It is.  I assume this is the result of my request.

What I would really like to know is how in large-memory applications we can completely bypass the static pool implementation and instead allocate our own CASESession objects.

Reply:
If we do like to bypass the pool implementation we can provide a new `CASEClientPoolDelegate` override.